plugins/libvirt/kvm_net

```diff
@@ -136,9 +137,28 @@ def get_vm_network_interface_names(pid):
         match = KVM_INTERFACE_NAME_REGEX.search(netdev_description)
         if match:
             result.add(match.groups()[0])
+    if os.getenv('ganeti'):
+        vm_name = _get_ganeti_vm_name(pid)
+        if vm_name:
+            for root, dirs, files in os.walk('/var/run/ganeti/kvm-hypervisor/nic/{0}'.format(vm_name)):
+               for name in files:
+                   with open(os.path.join(root, name)) as f:
+                       result.add(f.read())
     return result
 
 
+def _get_ganeti_vm_name(pid):
+    # grep -rl /var/run/ganeti/kvm-hypervisor/pid $PID
+    vm_name = ''
+    for root, dirs, files in os.walk('/var/run/ganeti/kvm-hypervisor/pid'):
+        for name in files:
+            with open(os.path.join(root, name)) as f:
+               read_pid = f.read()
+               if int(read_pid) == int(pid):
+                   vm_name = name
+                   break
+    return vm_name
+
```

Comment:
Please follow PEP8 and add two empty lines here.
`python3 -m flake8 FILENAME` is a good test for compliance.